### PR TITLE
i18n: Phrases for login page

### PR DESCRIPTION
**Pull request by tskarhed**
**What is this feature?**

Adds i18n markup for the login page. Some of the key names might be confusing, please leave suggestion if you have some better idea. I'll do the change/forgotten password in a separate PR.

**Special notes for your reviewer:**

For translations to be visible on the login page, the language needs to be set in the config
```
default_language = pseudo-LOCALE
``` 

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/locales/en-US/grafana.json [hunk 1]
```diff
@@ -1378,6 +1392,7 @@
     },
     "title": "Preferences"
   },
+  "Sign up": "Sign up",
```
Comment (tskarhed) [inline public/locales/en-US/grafana.json:1395]: Shouldn't this be under `login.signup.button-label`? 🤔

**Comment by tskarhed** [comment]
Running `yarn i18n:extract` on main produces warnings. There might be a CI related issue, if i18n stuff gets merged but not checked or updated.

**Comment by L-M-K-B** [comment]
@tskarhed There is an [open PR](https://github.com/grafana/grafana/pull/81571) for improving the i18n workflow. It will be merged until the end of this week. Afterwards your issues should be gone.

**Review by ashharrison90 — COMMENTED** [review]
lgtm! 🙌 

now we have the new process, i don't think you should have translations in the other language files. probably easiest thing to do is undo all the changes in the grafana.json files then rerun `yarn i18n:extract`. it should only modify the en-US phrases (and pseudo when committing)

**Review by ashharrison90 — APPROVED** [review]
looks great 🥳

**Pull request merged**